packages/opentelemetry-plugin-http/test/functionals/http-enable.test.ts

```diff
@@ -110,9 +108,9 @@ describe('HttpPlugin', () => {
       httpRequest
         .get(`http://${hostname}:${serverPort}${pathname}`)
         .then(result => {
-          const spans = audit.processSpans();
-          const outgoingSpan = spans[0];
-          const incomingSpan = spans[1];
+          const spans = spanProcessor.spans;
+          const outgoingSpan = spans[1];
+          const incomingSpan = spans[0];
```

Comment:
you can destructure instead of calling by index, like `const [incomingSpan, outgoingSpan] = spans;`